examples/web_agent/.gitignore

```diff
@@ -0,0 +1,163 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
I copied the standard gitignore for python projects and I added the extensions that we use.
I was taught to do this to avoid forgetting anything
I can change it to whatever is best